test/e2e/config/e2e_conf.yaml

```diff
@@ -140,3 +157,6 @@ intervals:
   default/wait-deployment: ["5m", "10s"]
   default/wait-job: ["5m", "10s"]
   default/wait-service: ["3m", "10s"]
+  kcp-remediation/wait-machines: ["5m", "10s"]
+  kcp-remediation/check-machines-stable: ["30s", "5s"]
+  kcp-remediation/wait-machine-provisioned: ["5m", "10s"]
```

Comment:
```suggestion
  rcp-remediation/wait-machines: ["5m", "10s"]
  rcp-remediation/check-machines-stable: ["30s", "5s"]
  rcp-remediation/wait-machine-provisioned: ["5m", "10s"]
```

Reply:
I don't think it is possible to rename these parameters, as i'm using the build-in capi test. This tests sets intervals like this:
https://github.com/kubernetes-sigs/cluster-api/blob/ef10e5aea3d3c9525dd83fa8a15005fc0b97d1b9/test/e2e/kcp_remediations.go#L151-L152

Where `specName` is hardcoded:
https://github.com/kubernetes-sigs/cluster-api/blob/main/test/e2e/kcp_remediations.go#L93